### DISCUSSION TITLE
can we get oil

### DISCUSSION THREAD

**Discussion by GTR7000** [voxeltycoon/issues#945] [General]
I want oil voxel tycoon

**Comment by DanielBarreleiro** [comment]
That's probably included in "Liquids" in the roadmap.
https://trello.com/c/3v2U7yhZ

Meanwhile there's a mod for Gasoline, Diesel...
https://steamcommunity.com/sharedfiles/filedetails/?id=2461470850

**Reply by GTR7000** [reply]
ok
